pkg/multiTenancyPlugins/quota/quotaMgmt.go

```diff
@@ -70,8 +70,8 @@ func (quota *QuotaMgmt) HandleCreateResponse(returnCode int, body []byte, tenant
 		log.Error(err)
 		return errors.New("Swarm Quota::HandleCreateResponse Create Container failed")
 	}
-	var containerConfig dockerclient.ContainerInfo
-	if err := json.NewDecoder(bytes.NewReader(body)).Decode(&containerConfig); err != nil {
+	var createResp types.ContainerCreateResponse
+	if err := json.NewDecoder(bytes.NewReader(body)).Decode(&createResp); err != nil {
 		//create response failed - container was NOT created on engine	-> decrease quota
 		err2 := quotaService.UpdateRQUsed(tenant, "", "", qp) //Decrease quota
```

Comment:
qp was short of previous quotaParam.
As we talked,R and all functions in quotaService like UpdateRQUsed were defined by Hai.